library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/AbstractPojoJoin.java

```diff
@@ -68,6 +76,11 @@ public AbstractPojoJoin(Class<?> outClass, String[] leftKeys, String[] rightKeys
     this.outClass = outClass;
   }
 
+  public AbstractPojoJoin(Class<?> outClass, String[] leftKeys, String[] rightKeys, Map<String, KeyValPair<STREAM, String>> outputToInputMap)
```

Comment:
Please add javadoc for this method. Also can you please add javadoc for the other constructors too?